lib/OpenQA/CacheService.pm

```diff
@@ -71,8 +71,9 @@ sub startup {
     $log->unsubscribe('message') if $ENV{OPENQA_CACHE_SERVICE_QUIET};
 
     # Increase busy timeout to 5 minutes
+    my $options = $ENV{OPENQA_CACHE_SERVICE_NO_WAL_MODE} ? '?no_wal=1' : '';
```

Comment:
How about `OPENQA_CACHE_SERVICE_MODE` which can have a value `wal` or `rollback` or `no_wal`?

Reply:
This is the underlying SQLite setting. https://www.sqlite.org/pragma.html#pragma_journal_mode

Reply:
Note that once WAL mode is enabled/disabled that's a persistent setting. So for `OPENQA_CACHE_SERVICE_MODE=delete` to work as expected, I would have to use `PRAGMA journal_mode=DELETE;` explicitly.

Reply:
ok, do that :) We all should know that the cache database is just a *cache* so safe to delete and recreate at the cost of some computational cost and time invested only